cpp/dolfinx/io/cells.cpp

```diff
@@ -274,6 +274,74 @@ std::vector<std::uint8_t> gmsh_quadrilateral(int num_nodes)
     throw std::runtime_error("Higher order GMSH quadrilateral not supported");
   }
 }
+//-----------------------------------------------------------------------------
+std::vector<std::uint8_t> discontinuous_triangle(int num_nodes)
```

Comment:
I don't follow how a cell can be discontinuous - a cell is just topological.

Reply:
The ordering of the degrees of freedom from a discontinuous Lagrange space are ordered in a lexicographical fashion, as all dofs are "internal" to the cell. This differs from how we order degrees of freedom from continuous Lagrange spaces. Therefore we need a permutation from the "discontinuous" ordering to the VTK output ordering